.github/workflows/DetectCDKBootstrapVersionChanges.yml

```diff
@@ -0,0 +1,32 @@
+name: Detect CDK Bootstrap Version Changes
+
+on: [pull_request]
```

Comment:
As far as I know, we can't. Mainly because these GitHub actions run on pull request or on push. Not sure if there is a scheduling mechanism and what that would look like. As for the internal pipeline, if there is something to be released, that means a PR was merged in which means the current check ran and passed. So I don't think we need to make a custom change to the internal pipelines.